modules/transcription-service-workflowoperation/src/main/resources/OSGI-INF/amberscript-attach-transcription.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Xml file formated differently compared to other xml files in the same application

Reply:
fixed